### PR TITLE
Standard sample template for wiki layout

### PR DESCRIPTION
**Pull request by JosiahWI**
One of the concerns about our choice of AsciiDoc as a format was that it's not as well known as Markdown. Having a good template should help new contributors get involved, and will also help keep things consistent across the wiki. The layout is still very undecided, but this is a humble beginning that we can work on.

### CODE REVIEW

**Review by appgurueu — CHANGES_REQUESTED** [review]
2.33 MB for the `pomegranate.png` seems slightly wasteful to me :sweat_smile:

**Comment by rollerozxa** [comment]
Is there a slightly smaller example image we could use?

File: templates/standard.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+So without further ado, here is what we should see when we are done.
+
+[#img-pomegranate]
+.The Expected Result
+image::pomegranate.png[Pomegranate]
+
+== Pomegranate API Methods
+
+* `plant_tree()`
```
Comment (benrob0329) [inline templates/standard.adoc:29]:
```suggestion
=== `plant_tree(pos, name)`
* `pos`: Vector
* `name`: string

Plants a tree of type `name` at `pos` if `name` is a registered pomegranate tree.

.Planting A Tree
[example, lua]
---
plant_tree(Vector.new(35, 42, 36), "regular")
---

```

File: templates/standard.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* `grow_tree()`
```
Comment (benrob0329) [inline templates/standard.adoc:30]:
```suggestion
=== `grow_tree(pos)`
* `pos`: Vector

Grows the tree at `pos` if the node at `pos` is a valid pomegranate tree.

```

File: templates/standard.adoc [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+* `spray_tree()`
```
Comment (benrob0329) [inline templates/standard.adoc:31]:
```suggestion
=== `spray_tree(pos, pesticide)`
* `pos`: Vector
* `pesticide`: string

Sprays tree at `pos` with `pesticide` if it is a registered chemical.
```

**Review by benrob0329 — CHANGES_REQUESTED** [review]

File: templates/standard.adoc (hunk 1 above)
Comment (appgurueu) [reply templates/standard.adoc:28]: Not happy with the `Vector`. Should be lowercase and monospaced.

Comment (JosiahWI) [reply templates/standard.adoc:28]: Agreed.

Comment (benrob0329) [reply templates/standard.adoc:28]: `Vector` is a class name in this examples (like it is in MT), but I agree with it being monospaced.

Comment (JosiahWI) [reply templates/standard.adoc:28]: Can you add the monospace? And then I will commit the suggestions.

Comment (benrob0329) [reply templates/standard.adoc:28]: I already made `Vector` monospaced, also make `string` monospaced.

File: templates/standard.adoc [hunk 4]
```diff
@@ -0,0 +1,52 @@
+= Pomegranates
+Author Minetest-Docs and Contributors
```
Comment (appgurueu) [inline templates/standard.adoc:2]: Why do we need this? Git already conveniently stores this info for us.

**Review by appgurueu — CHANGES_REQUESTED** [review]

File: templates/standard.adoc [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+* `pos`: `vector`
+* `name`: `string`
+
+Plants a tree of type `name` at `pos` if `name` is a registered pomegranate tree.
+
+.Planting A Tree
+[example, lua]
+---
+plant_tree(vector.new(35, 42, 36), "regular")
+---
```
Comment (benrob0329) [inline templates/standard.adoc:38]:
```suggestion
====
[source, lua]
---
plant_tree(vector.new(35, 42, 36), "regular")
---
====
```

Example delimiters let you use blocks inside of it.

Comment (benrob0329) [reply templates/standard.adoc:38]: https://docs.asciidoctor.org/asciidoc/latest/blocks/example-blocks/#delimited

File: templates/standard.adoc (hunk 4 above)
Comment (JosiahWI) [reply templates/standard.adoc:2]: It's required for us to also have the revision number, which I think is still useful while reading the documentation.

File: templates/standard.adoc [hunk 6]
```diff
@@ -29,7 +28,8 @@ image::pomegranate.png[Pomegranate]
 * `pos`: `vector`
 * `name`: `string`
 
-Plants a tree of type `name` at `pos` if `name` is a registered pomegranate tree.
+Plants a tree of type `name` at `pos`
```
Comment (appgurueu) [inline templates/standard.adoc:31]: That seems excessive to me

Comment (benrob0329) [reply templates/standard.adoc:31]: Perhaps, but we can change it if a better syntax is found.

File: templates/standard.adoc (hunk 4 above)
Comment (benrob0329) [reply templates/standard.adoc:2]: This will get moved into a header anyways here shortly anyways

**Review by benrob0329 — APPROVED** [review]

**Pull request merged**